src/state.rs

```diff
@@ -292,6 +268,39 @@ impl State {
         }
     }
 
+    pub fn active_project(&mut self) -> Result<&mut RemoteCapellaProject, ShellError> {
```

Comment:
Because when we do the fetching of the project ID we set the id field of the `RemoteCapellaProject` so it needs to be mutable. The same applies for the organization.